src/main/scala/shine/DPIA/Compilation/TranslationToImperative.scala

```diff
@@ -131,4 +131,27 @@ object TranslationToImperative {
     }
   }
 
+  def str(E: Phrase[ExpType])
```

Comment:
The name is confusing because it seems like it is a function that has to do with `strings`.